pyproject.toml

```diff
@@ -47,6 +47,7 @@ dynamic = ['version']
 all = [
     'phonenumbers>=8,<9',
     'pycountry>=22,<23',
+    'schwifty>=2023.03.0,<=2023.03.0'
```

Comment:
cc @yezz123 @Kludex
What are our guidelines to plug a version on pyproject.toml ? is there a CI workflow that will try more recent versions of our dependencies ? I think we should extend this to a broader range of versions.

Reply:
```suggestion
    'schwifty>=2023.03.0,<=2024.03.0'
```

Currently, you can do something like this it will be much better

Reply:
Done